packages/instrumenter/src/mutators/array-declaration-mutator.ts

```diff
@@ -0,0 +1,30 @@
+import { NodePath, types } from '@babel/core';
+
+import { NodeMutation } from '../mutant';
+
+import { NodeMutator } from './node-mutator';
+
+export class ArrayDeclarationMutator implements NodeMutator {
+  public name = 'ArrayDeclaration';
+
+  public mutate(path: NodePath): NodeMutation[] {
+    const currentNode = path.node;
+
+    if (types.isArrayExpression(currentNode)) {
+      const replacement = currentNode.elements.length ? types.arrayExpression() : types.arrayExpression([types.stringLiteral('Stryker was here')]);
+      return [{ original: currentNode, replacement }];
+    } else if (
+      (types.isCallExpression(currentNode) || types.isNewExpression(currentNode)) &&
```

Comment:
Why not use `path.isCallExpression()` and `path.isNewExpression()`, etc

Reply:
Fixed!